_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -408,6 +408,23 @@ public function get_all_options() {
 
 		foreach ( $settings as $setting => $properties ) {
 			switch ( $setting ) {
+				case 'lang_id':
+					if ( defined( 'WPLANG' ) ) {
+						// We can't affect this setting, so warn the client
+						$response[ $setting ] = 'error_const';
+						break;
+					}
+
+					if ( ! current_user_can( 'install_languages' ) ) {
+						// The user doesn't have caps to install language packs, so warn the client
+						$response[ $setting ] = 'error_cap';
+						break;
+					}
+
+					$value = get_option( 'WPLANG' );
+					$response[ $setting ] = empty( $value ) ? 'en_US' : $value;
```

Comment:
According to this, we'll return `en_US` as a default language, but according to the definition of the setting in `get_updateable_data_list()`, it's be expected that it would return an empty string. Perhaps we should use the same in both locations?

Reply:
Explicitly set it to `en_US` in the declaration in 2a408ec. Thanks!